engine/verification/fetcher/engine.go

```diff
@@ -393,55 +396,45 @@ func (e *Engine) validateChunkDataPack(chunkIndex uint64,
 	}
 
 	// 3. collection id must match
-	err := e.validateCollectionID(collection, chunkDataPack, result, chunk)
+	err := e.validateCollectionID(chunkDataPack, result, chunk)
 	if err != nil {
-		return fmt.Errorf("could not validate collection: %x, from sender ID: %x, block ID: %x, resultID: %x, chunk ID: %x",
-			collection.ID(),
-			senderID,
-			blockID,
-			result.ID(),
-			chunk.ID())
```

Comment:
Redundant information: all these are appended to the error at the higher levels of these nested calls.